src/components/ProductInfo/ChangeProductForm/ChangeProductForm.tsx

```
@@ -0,0 +1,378 @@
+      <form
+        onSubmit={(e) => {
+          e.preventDefault();
+          e.stopPropagation();
+          form.handleSubmit().then();
+        }}
+      >
+        <form.Field
+          name="fullName"
+          children={(field) => {
+            return (
+              <div className={s.fieldWrapper}>
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)